tools/cobo/test/Makefile.in

```diff
@@ -1,9 +1,8 @@
-# Makefile.in generated by automake 1.11.1 from Makefile.am.
+# Makefile.in generated by automake 1.10.1 from Makefile.am.
```

Comment:
Oops. I think this `Makefile.in` is a deadwood. I actually see your later commit to remove this (a2855a7). Nice work!